src/metabase/search/postgres/index.clj

```diff
@@ -50,7 +50,12 @@
              [:database_id :int]
              [:table_id :int]
              ;; filter related
-             [:archived :boolean]])
+             [:archived :boolean]
+             ;; useful for tracking the speed and age of the index
```

Comment:
Pretty useful for making sure indexing is working 😆 

Another nice use of it is to create as stats dashboard about how long indexing takes.